### PR TITLE
Fixes #29779: Drop passenger packages

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: comps/comps-foreman-rhel7.xml [hunk 1]
```diff
@@ -240,9 +239,6 @@
       <packagereq type="default">tfm-rubygem-ovirt-engine-sdk</packagereq>
       <packagereq type="default">tfm-rubygem-paint</packagereq>
       <packagereq type="default">tfm-rubygem-parallel</packagereq>
-      <packagereq type="default">tfm-rubygem-passenger</packagereq>
-      <packagereq type="default">tfm-rubygem-passenger-native</packagereq>
-      <packagereq type="default">tfm-rubygem-passenger-native-libs</packagereq>
```
[the remark this answers is not in the document]
Comment (ehelms) [reply comps/comps-foreman-rhel7.xml:244]: I've included the addition of these to the Obsoletes in tfm.

File: packages/foreman/tfm/tfm.spec [hunk 2]
```diff
@@ -131,6 +131,12 @@ Obsoletes: tfm-ror52-rubygem-turbolinks-doc < 2.5.4-3
 Obsoletes: tfm-ror52-rubygem-arel <= 9.0.0
 Obsoletes: %{scl_prefix}-rubygem-arel <= 9.0.0
 
+# obsolete all of the tfm Passenger packages since Passenger support is dropped
```
Comment (ekohl) [inline packages/foreman/tfm/tfm.spec:134]: I think it should also do the same with doc packages but not entirely sure.

Comment (ehelms) [reply packages/foreman/tfm/tfm.spec:134]: Done.

**Review by ekohl — APPROVED** [review]
Code looks good. Not merging yet since that's probably good to coordinate a bit.

**Pull request merged**